modules/cluster-logging-upgrading-logging-from-44.adoc

```diff
@@ -0,0 +1,304 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-upgrading.adoc
+
+[id="cluster-logging-updating-logging-from-44_{context}"]
+= Updating cluster logging from 4.4
+
+After updating the {product-title} cluster, you can update cluster logging from 4.4 to 4.6 by changing the subscription for the Elasticsearch Operator and the Cluster Logging Operator.
+
+When your cluster logging version is 4.4, you must update cluster logging as follows due to changes in the way that OLM provides operator packages.
+
+Additionally, it is recommended that you update Cluster Logging Operator before you update Elasticsearch Operator to avoid
+the need to restart the each Elasticsearch pod manually, one by one.
+
+Perform each update version by version in the following order:
```

Comment:
Add punctuation? Maybe: Perform each update, version by version, in the following order:

Reply:
Done.